frontend/public/components/factory/list-page.jsx

```diff
@@ -59,17 +59,20 @@ export const TextFilter = (props) => {
 TextFilter.displayName = 'TextFilter';
 
 // TODO (jon) make this into "withListPageFilters" HOC
-/** @augments {React.PureComponent<{ListComponent: React.ComponentType<any>, kinds: string[], filters?:any, flatten?: function, data?: any[], rowFilters?: any[], hideToolbar?: boolean, hideLabelFilter?: boolean }>} */
+/** @augments {React.PureComponent<{ListComponent: React.ComponentType<any>, kinds: string[], filters?:any, flatten?: function, data?: any[], rowFilters?: any[], hideToolbar?: boolean, hideLabelFilter?: boolean, hideColumnFilter?: boolean, columnFilters?: any }>} */
 export class ListPageWrapper_ extends React.PureComponent {
   render() {
     const {
+      Header,
       flatten,
       ListComponent,
       reduxIDs,
       rowFilters,
       textFilter,
       hideToolbar,
       hideLabelFilter,
+      hideColumnFilter,
+      columnFilters,
```

Comment:
Again, I don't think these are really filters. Maybe `selectedColumns`.

Reply:
updated to selectedColumns